install.sh

```diff
@@ -1037,7 +1037,26 @@ download_latest_bundle_and_extract_if_needed() {
     rm -rf $BUILD_DIR
     local URL="https://dl.sandstorm.io/sandstorm-$BUILD.tar.xz"
     echo "Downloading: $URL"
-    curl -A "$CURL_USER_AGENT" -f "$URL" | tar Jxo
+    curl -A "$CURL_USER_AGENT" -f "$URL" > /var/tmp/sandstorm-$BUILD.tar.xz
```

Comment:
`/var/tmp/` is a shared namespace, so another user on the same machine could place their own content at `/var/tmp/sandstorm-$BUILD.tar.xz`, or over the .sig or even the keyring files.

I'd suggest making a folder under `/var/tmp` with something like:

```
mktemp --directory --tmpdir=/var/tmp -t sandstorm-install.XXXXXXXX
```

and then doing all the other work under that folder.